starcheck/src/aca_load_review_cl.rst

```diff
@@ -208,10 +208,10 @@ Checks
 |       |                  | | |configuration (2    |      |     |    |    |    |                |
 |       |                  | | |monitor windows)    |      |     |    |    |    |                |
 +-------+------------------+-+-+--------------------+------+-----+----+----+----+----------------+
-|ACA-009|Magnitude limit   |X|X|AS: 5.8 - 10.3 (or fainter, if needed to   |n/a |Possible Bright |
+|ACA-009|Magnitude limit   |X|X|AS: 5.2 - 10.3 (or fainter, if needed to   |n/a |Possible Bright |
 |       |                  | | |find stars)                                |    |Star Hold       |
 +-------+------------------+-+-+-------------------------------------------+----+----------------+
-|ACA-010|Magnitude limit   |X|X|GS: 6.0 - 10.3 (or fainter, if needed to   |n/a |Reduced aspect  |
+|ACA-010|Magnitude limit   |X|X|GS: 5.4 - 10.3 (or fainter, if needed to   |n/a |Reduced aspect  |
```

Comment:
I've changed my mind.  I think here in the checklist and in the code that the "spit out a warning" limit should just be 5.2 for guide stars too and the star selection code should handle subtleties around it.